main/src/ca/uwaterloo/flix/util/ParOps.scala

```diff
@@ -69,14 +77,45 @@ object ParOps {
     */
   @inline
   def parAgg[A, S](xs: Iterable[A], z: => S)(seq: (S, A) => S, comb: (S, S) => S)(implicit flix: Flix): S = {
```

Comment:
Impressive that you got this to work.

Reply:
I actually wanted something very simple, e.g. based on https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/RecursiveTask.html

(I should have stated that of course).

Reply:
Once you have tried `RecursiveTask`, we can compare your hand-rolled implementation to Java state-of-the-art :) (pre VTs).

Reply:
I'm having trouble imagining a recursive version that would be efficient.

Reply:
What do you have in mind?

Reply:
@mlutze @JonathanStarup Any idea how to rewrite this function using plain Java primitives?